bin/node-template/node/Cargo.toml

```diff
@@ -23,6 +23,7 @@ futures = { version = "0.3.21", features = ["thread-pool"]}
 sc-cli = { version = "0.10.0-dev", path = "../../../client/cli" }
 sp-core = { version = "7.0.0", path = "../../../primitives/core" }
 sc-executor = { version = "0.10.0-dev", path = "../../../client/executor" }
+sc-network-common = { version = "0.10.0-dev", path = "../../../client/network/common" }
```

Comment:
All types required for sc-service configuration should rather be re-exported by sc-service.